### PR TITLE
Verify metadata before uploading to pypi.

### PR DESCRIPTION
**Pull request by cqc-alec**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -394,8 +394,6 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_PYTKET_API_TOKEN }}
         verbose: true
-        # TODO Remove this line when long_description added:
-        verify_metadata: false
```
Comment (cqc-melf) [inline .github/workflows/release.yml:398]: I assume that this is automatically set to true?

**Review by cqc-melf — APPROVED** [review]
Is this also checked for all the extensions?

**Comment by cqc-alec** [comment]
> Is this also checked for all the extensions?

Yes, the extensions have this check.

File: .github/workflows/release.yml (hunk 1 above)
Comment (cqc-alec) [reply .github/workflows/release.yml:398]: Yes: https://github.com/pypa/gh-action-pypi-publish#disabling-metadata-verification

**Pull request merged**